### DISCUSSION TITLE
肉鸽选刷源石锭时，商店可以逻辑优化一下

### DISCUSSION THREAD

**Discussion by MuNitCat** [MaaAssistantArknights/MaaAssistantArknights#3847] [Ideas]
现在肉鸽选择**多刷源石**锭的选项（刷到第二层就退）时，在商店的逻辑是在投资完后，剩下的源石锭还会去买东西，这里操作花的时间也较多
商店的逻辑在**多刷源石锭**的情况下优化成投资完毕就直接退出可以省一点时间

**Comment by StardustOvO-000** [comment]
应该是考虑到大多数人奖励没拿满，买藏品可以获得更多的结算分数
